python-avd/pyavd/_eos_designs/shared_utils/wan.py

```diff
@@ -501,14 +500,18 @@ def wan_ha_peer(self: SharedUtilsProtocol) -> str | None:
         raise AristaAvdError(msg)
 
     @cached_property
-    def vrf_default_uplinks(self: SharedUtilsProtocol) -> list:
-        """Return the uplinkss in VRF default."""
-        return [uplink for uplink in self.get_switch_fact("uplinks") if get(uplink, "vrf") is None]
+    def vrf_default_uplinks(self: SharedUtilsProtocol) -> EosDesignsFactsProtocol.Uplinks:
+        """
+        Return the uplinkss in VRF default.
+
+        TODO: Figure out if we really need this, since all uplinks are in vrf default.
```

Comment:
I think it may be a leftover of an attempt to also add VRF on uplinks while working on WAN stuff we can probably get rid of this in another PR and we will bring it back when/if we need

Reply:
I will leave the TODO here so we don't forget.